fjord/base/tests/test_google_utils.py

```diff
@@ -0,0 +1,101 @@
+from functools import wraps
+import sys
+
+from mock import MagicMock, patch
+from nose.tools import eq_
+import requests_mock
```

Comment:
I thought all the `import xxx` were supposed to go above the `from xxx import xxx`. /me looks up webdev bootcamp